CHANGELOG.md

```diff
@@ -3,7 +3,82 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.43.0
+
+This release contains a number of fixes and enhancements.
+
+### Tooling, SDK, and Runtime
+
+- Add line count to test coverage report (#4843) (authored by @FarisR99)
+- build: run 'make generate' in release container
+- Bundles fixes, changes:
+  - plugins: accept any HTTP 2xx for status and logs (#4897) (authored by @lvisterin)
+  - runtime: Treat other Unix-es like macos (#4931) (authored by @iamleot)
```

Comment:
A little description will help.

Reply:
Right. I think the patch-generator script just chucks over the exact commit message. Some edits for cleanup+consistency could go a long way here.

Reply:
I have reworded/rewritten the descriptions for all of the PRs to better reflect what changed.